test/user.recommendations.test.js

```diff
@@ -14,35 +16,44 @@ describe('Recommendations', () => {
     return await request(app)
     .get('/api/v1/users/recommendations/44a4e0cd5616b584e6727171016f37bc943d456cb4d4f4f94ca3ff8def77da64')
     .then(response => {
-      (response.body)
       expect(response.statusCode).toBe(200)
-      expect(Object.keys(response.body[0])).toContain('name')
-      expect(Object.keys(response.body[0])).toContain('url')
-      expect(Object.keys(response.body[0])).toContain('hours')
+      expect(Object.keys(response.body.locations[0])).toContain('name')
+      expect(Object.keys(response.body.locations[0])).toContain('url')
+      expect(Object.keys(response.body.locations[0])).toContain('hours')
     })
   });
 
-  // it("POST request for specific recommendation", async () => {
-  //   return await request(app)
-  //   .post('/api/v1/users/recommendations/44a4e0cd5616b584e6727171016f37bc943d456cb4d4f4f94ca3ff8def77da64?yelpId=UeIXzDWs8YLusMDYElJuSQ')
-  //   .then(response => {
-  //     expect(response.statusCode).toBe(201)
-  //     // expect(Object.keys(response.body[0])).toContain('name')
-  //     // expect(Object.keys(response.body[0])).toContain('url')
-  //     // expect(Object.keys(response.body[0])).toContain('hours')
-  //   })
-  // });
-
-  // it("GET request for all specific user's recommendations", async () => {
-  //   return await request(app)
-  //   .get('/api/v1/users/recommendations/44a4e0cd5616b584e6727171016f37bc943d456cb4d4f4f94ca3ff8def77da64')
-  //   .then(response => {
-  //     (response.body)
-  //     expect(response.statusCode).toBe(200)
-  //     expect(Object.keys(response.body[0])).toContain('name')
-  //     expect(Object.keys(response.body[0])).toContain('url')
-  //     expect(Object.keys(response.body[0])).toContain('hours')
-  //   })
-  // });
+  it("POST request to create a user recommendation", async () => {
```

Comment:
I appreciate seeing another way of testing here, though it's subtly different. I'd say though that we should strive to maintain similarity between all tests going forward.